install_files/securedrop-app-code/debian/postinst

```diff
@@ -38,27 +38,27 @@ database_migration() {
     cd '/var/www/securedrop/'
 
     # `alembic current` outputs nothing if the DB isn't under version control.
-    if [ -z "$(${SDBIN}/alembic current)" ]; then
+    if [ -z "$(sudo -u www-data ${SDBIN}/alembic current)" ]; then
         sqlite_tables=$(sqlite3 "$database_file" .tables)
 
         if ! echo "$sqlite_tables" | grep -q journalists; then
             # This is a fresh install, run the full migrations.
-            ${SDBIN}/alembic upgrade head
+            sudo -u www-data ${SDBIN}/alembic upgrade head
         else
             # This is an old install, so mark that the schema is up to date
             # as of the version of migration 15ac9509fc68_init.py when we first
             # started version controlling the database.
-            ${SDBIN}/alembic stamp 15ac9509fc68
+            sudo -u www-data ${SDBIN}/alembic stamp 15ac9509fc68
```

Comment:
running `dpkg-reconfigure` to hit this line in the postinst regenerates `/var/www/securedrop/__pycache__/config.cpython-38.pyc` with `www-data` the owner.

If I run a downgrade and then upgrade again, both as `www-data`, it also recompiles config.py.